promptos/system_prompts/02_scribe.md

```
@@ -0,0 +1,147 @@
+
+### Prerequisites
+- Node.js 18+ ([install guide](https://nodejs.org))
+- Git ([install guide](https://git-scm.com))
+
+### Quick Install (Recommended)
+```bash
+npx create-awesome-app my-project
+cd my-project
+npm start
+```
+✨ Your app is now running at http://localhost:3000
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)